entities/template.go

```diff
@@ -20,3 +20,8 @@ type TemplateMeta struct {
 	Name      string    `json:"name"`
 	Timestamp time.Time `json:"timestamp"`
 }
+
+type TemplatesCollection struct {
+	Name        string `json:"name"`
+	SubjectPart string `json:"subject_part"`
```

Comment:
This needs to include the timestamps as well. Also since this is a new entity, will this work with our `Paginate` function? I think this relies on having the `Model` interface methods, but I'm not sure, we'll need to properly test this.

Reply:
@FilipNikolovski Yeah you are right there must be the id because if we want the pagination to work we need to implement `ModelInterface`. You can check the changes.